src/main/java/seedu/duke/data/TransactionList.java

```diff
@@ -326,16 +325,18 @@ public HashMap<String, int[]> processMonthlyExpenditure(HashMap<String, int[]> m
      * @return A string containing the comment related to the spending habit for the month.
      */
     public String getSpendingHabitComment(int income, int savings) {
-        int savingsPercentage = HUNDRED_PERCENT * savings / income;
-
-        if (savingsPercentage >= HUNDRED_PERCENT) {
-            return INFO_STATS_HABIT_VERY_HIGH_SAVINGS.toString();
-        } else if (savingsPercentage >= SEVENTY_FIVE_PERCENT) {
-            return INFO_STATS_HABIT_HIGH_SAVINGS.toString();
-        } else if (savingsPercentage >= FIFTY_PERCENT) {
-            return INFO_STATS_HABIT_MEDIUM_SAVINGS.toString();
-        } else if (savingsPercentage >= TWENTY_FIVE_PERCENT) {
-            return INFO_STATS_HABIT_LOW_SAVINGS.toString();
+        if (income > MIN_AMOUNT_VALUE) {
```

Comment:
MIN_AMOUNT_VALUE is 0.   Income cannot be zero, as it would be indivisible and would throw an arithmetic error.